.github/workflows/merge-commits-to-next-branch.yml

```diff
@@ -0,0 +1,133 @@
+name: Merge commits to next branch
+
+on:
+  schedule:
+    - cron: "0 */2 * * *"
+
+env:
+  COMMIT_INFO: commit-info
+
+jobs:
+  dequeue:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Azure Storage
+        run: npm i -g @azure/storage-queue
+      - name: Check queue first item
+        uses: actions/github-script@v6
+        with:
+          script: |
+              const { QueueClient, QueueServiceClient } = require("@azure/storage-queue");
+              const connectionString = `${{ secrets.AZURE_STORAGE_CONNECTION_STRING }}`;
+              const queueName = `${{ secrets.AZURE_QUEUE_NAME }}`;
+              const queueServiceClient = QueueServiceClient.fromConnectionString(connectionString);
+              const queueClient = queueServiceClient.getQueueClient(queueName);
+              var receivedMessages = await queueClient.receiveMessages();
+              const message = receivedMessages.receivedMessageItems[0];
+              $COMMIT_INFO = JSON.parse(message.messageText);
+  create-branch:
+    runs-on: ubuntu-latest
+    if: ( $COMMIT_INFO.label == 'queued' )
+    steps:
+      - name: Create branch
+        uses: actions/github-script@v6
+        with:
+          script: |
+            git fetch origin
+            git checkout test_next
+            git checkout -b main-next-$COMMIT_INFO.sha
+            git merge $COMMIT_INFO.sha --no-ff
+  create-bug:
```

Comment:
If there are no merge conflicts, we can open a new pull request to see if CI checks pass. but if merge conflicts exist, we can log an issue/bug